BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Chris/ChrisViewController.swift

```diff
@@ -124,27 +71,62 @@ extension ChrisViewController {
         myButton.layer.shadowRadius  = 4.0
         myButton.layer.shadowOffset  = CGSize(width: 2.0, height: 2.0)
         
-        
         view.addSubview(myButton) // <--- Add it to the view
         
-        
-        
-        
-        
-        
-        
-        
-        
-        // Or you can create Contraints for the UIButton 👇
+        // You can also create Contraints for the UIButton 👇
         myButton.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint(item: myButton, attribute: .width, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: 170).isActive = true
         NSLayoutConstraint(item: myButton, attribute: .height, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: 44).isActive = true
         NSLayoutConstraint(item: myButton, attribute: .centerX, relatedBy: .equal, toItem: view, attribute: .centerX, multiplier: 1, constant: 0).isActive = true
         NSLayoutConstraint(item: myButton, attribute: NSLayoutConstraint.Attribute.top, relatedBy: .equal, toItem: storyboardButton, attribute: .bottom, multiplier: 1, constant: 50).isActive = true
-        
     }
     
+    @objc func createAnotherButton() {
+        
+        // You can also create a button and assing a type right away using the convenience init
+        let anotherButton = UIButton(type: .custom)
+        anotherButton.layer.borderColor = UIColor.blue.cgColor
+        anotherButton.layer.borderWidth = 1.0
+        anotherButton.setTitle("New Button", for: .normal)
+        anotherButton.setTitleColor(.blue, for: .normal)
+        
+        //To add an Image to the button use setImage function
+        let swiftMiamiLogo = UIImage(named: "SwiftMiamiLogo")
+        anotherButton.setImage(swiftMiamiLogo, for: .normal)
+        
+        //To adjust the position of the Image and Title you assign imageEdgeInsets & titleEdgeInsets vars.
+        anotherButton.imageEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 140)
+        anotherButton.titleEdgeInsets = UIEdgeInsets(top: 10, left: -970, bottom: 10, right: 10)
+        
+        //Finally to manage content alignment you set contentHorizontalAlignment & contentVerticalAlignment vars
+        anotherButton.contentHorizontalAlignment = .left
+        anotherButton.contentVerticalAlignment = .center
+        
+        view.addSubview(anotherButton)
+        
+        // Contraints
+        anotherButton.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint(item: anotherButton, attribute: .width, relatedBy: .equal, toItem: nil, attribute: .notAnAttribute, multiplier: 1, constant: 170).isActive = true
```

Comment:
This is an alternative way of expressing these constraints, which is a little bit less verbose:

```swift
anotherButton.widthAnchor.constraint(equalToConstant: 170).isActive = true
anotherButton.heightAnchor.constraint(equalToConstant: 44).isActive = true
anotherButton.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
anotherButton.topAnchor.constraint(equalTo: storyboardButton.bottomAnchor, constant: 250).isActive = true
```
😄